build.sbt

```diff
@@ -21,4 +21,6 @@ lazy val intellijHaskell = (project in file(".")).
     unmanagedJars in Compile += baseDirectory.value / "idea"
   )
 
-ideaBuild in ThisBuild := "163.7743.44"
\ No newline at end of file
+ideaBuild in ThisBuild := "163.7743.44"
+
+javacOptions ++= Seq("-encoding", "UTF-8")
```

Comment:
@igrep Can you fix this with changing the default charset in Windows Terminal? IIRC the default encoding in Windows Console is not UTF-8.